queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2019-01-01';
```

Comment:
| Requirements 📝 |
| --- |
| ![image](https://github.com/kwesiObuobi/vet-clinic/assets/93039359/c16ce0fc-53b2-45dc-ba3b-2db14580c08a) |

- [ ] It would be nice if you follow the requirements because this query should return the records between `2016-01-01` and `2019-12-31` because even if you don't write the day and month and just write `BETWEEN 2016 and 2019` then the query will include the whole 2019 as well 👍

Reply:
Ohh thanks again
I totally missed that.

Reply:
well resolved! ✔️